# 
# Submitted-By: Robert Schwebel, 2005-12-14
# Committed-By: Robert Schwebel
#
# Error: 
#
# Sometimes CONTROL files are being archived by accident; the -X
# argument needs to come before the file argument. 
#
# Description: 
#
# State:
# 
# undefined
#

diff -urN ipkg-utils-1.7/ipkg-build ipkg-utils-1.7-ptx/ipkg-build
--- ipkg-utils-1.7/ipkg-build	2004-04-24 11:43:51.000000000 +0200
+++ ipkg-utils-1.7-ptx/ipkg-build	2005-12-14 23:30:52.000000000 +0100
@@ -226,7 +226,7 @@
 mkdir $tmp_dir
 
 echo $CONTROL > $tmp_dir/tarX
-( cd $pkg_dir && tar $ogargs -czf $tmp_dir/data.tar.gz . -X $tmp_dir/tarX )
+( cd $pkg_dir && tar $ogargs -X $tmp_dir/tarX -czf $tmp_dir/data.tar.gz . )
 ( cd $pkg_dir/$CONTROL && tar $ogargs -czf $tmp_dir/control.tar.gz . )
 rm $tmp_dir/tarX
 
